modules/shared/components/organisms/ImagePollView/ImagePollView.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import PostViewHeader from '../../molecules/PostViewHeader/PostViewHeader';
+import PostViewFooter from '../../molecules/postFooter/PostFooter';
+import type { IImagePollView } from './IImagePollView';
+import ImagePollGroup from '../../molecules/ImagePollGroup/ImagePollGroup';
+
+const ImagePollView: FC<IImagePollView.IProps> = ({ post }): ReactElement => {
+  const firstIndex = 0;
+  return (
+    <div className="bg-white p-m shadow-soft rounded-md space-y-4" id={post.id}>
+      <PostViewHeader
+        id={post.id}
+        date={new Date(post.created_at)}
+        name={post.user.name}
+        profilePic={post.user.profile_pic}
+        isHidden={post.is_hidden}
```

Comment:
`isHidden`, what will be `hidden` ( the post, the post date, post title,... )
it may be confusing for newcomers, also the code will be self-documented if we are more explicit.